terraform-aws/coordinator.tf

```diff
@@ -71,7 +71,7 @@ resource "aws_elb" "coordinator-lb" {
     [aws_security_group.presto.id],
     var.additional_security_groups,
   )
-  subnets  = [var.subnet_id]
```

Comment:
The refactoring of the subnets happened because this fails if there is only one subnet.

It was unclear from the documentation and usage whether this variable was supposed to hold one subnet or many.

Reply:
I haven't yet updated the documentation with my changes to the variables, by the way.